src/server/utilities/cspHeader/index.test.js

```diff
@@ -41,533 +41,535 @@ describe('cspHeader', () => {
   afterAll(() => {
     delete process.env.SIMORGH_APP_ENV;
   });
+  'https://publish.flourish.studio', // Flourish embeds
```

Comment:
I really don't understand this massive whitespace change?

If you enable hide whitespace above you can see the actual change better:
![image](https://github.com/bbc/simorgh/assets/9645462/b1f20883-a895-4f64-a7c4-7e3673c7dace)